tests/inventory/pipelines/load_groups_pipeline_test.py

```diff
@@ -49,26 +49,15 @@ def setUp(self):
                 self.mock_admin_client,
                 self.mock_dao))
 
-    @mock.patch.object(load_groups_pipeline, 'metadata_server')
-    def test_can_inventory_google_groups(self, mock_metadata_server):
+    def test_can_inventory_google_groups(self):
         """Test inventory groups can be configured per config values."""
-
-        # GCP case, configs are good.
-        mock_metadata_server.can_reach_metadata_server.return_value = True
-        self.assertTrue(self.pipeline._can_inventory_google_groups())
-
-        # Local case, configs are good.
-        mock_metadata_server.can_reach_metadata_server.return_value = False
         self.assertTrue(self.pipeline._can_inventory_google_groups())
 
-        # GCP case, configs are bad.
-        mock_metadata_server.can_reach_metadata_server.return_value = True
-        self.pipeline.required_gcp_execution_config = ['aaaaa', None]
-        self.assertFalse(self.pipeline._can_inventory_google_groups())
-
-        # Local case, configs are bad.
-        mock_metadata_server.can_reach_metadata_server.return_value = False
-        self.pipeline.required_local_execution_config = ['aaaaa', None, 'bbbbb']
+    def test_cannot_inventory_google_groups(self):
+        """Test inventory groups cannot be configured per config values."""
+        self.pipeline.configs['groups_service_account_email'] = None
```

Comment:
This is cool, I should've seen the seam here to change the value for testing.  :)